.github/actions/maven-build-and-tag/action.yml

```diff
@@ -0,0 +1,148 @@
+name: Build and tag Maven project
+description: Build and tag Maven project
+inputs:
+  java-version:
+    description: The Java version to perform the build
+    required: false
+    default: '11'
+  java-distribution:
+    description: The Java distribution to perform the build
+    required: false
+    default: 'temurin'
+  property-to-update:
+    description: Property to update in addition to the version of the pom file
+    required: false
+  extra-maven-opts:
+    description: Extra maven options for the build command line
+    required: false
+  nexus-username:
+    description: Nexus user name
+    required: true
+  nexus-password:
+    description: Nexus password
+    required: true
```

Comment:
maybe rename to maven since nexus it's the name of the product we are using for hosting maven repos

Reply:
That would make sense but right now there is more convention towards nexus than maven naming...:
```
$ git grep nexus-username 
.github/actions/maven-build-and-tag/action.yml:  nexus-username:
.github/actions/maven-build-and-tag/action.yml:        MAVEN_USERNAME: ${{ inputs.nexus-username }}
.github/actions/maven-build-and-tag/action.yml:        MAVEN_USERNAME: ${{ inputs.nexus-username }}
.github/actions/maven-release/action.yml:  nexus-username:
.github/actions/maven-release/action.yml:        NEXUS_USERNAME: "${{ inputs.nexus-username }}"
.github/actions/nexus-create-staging/action.yml:  nexus-username:
.github/actions/nexus-create-staging/action.yml:        NEXUS_USERNAME: ${{ inputs.nexus-username }}
.github/workflows/build-and-tag-maven.yml:          nexus-username: ${{ secrets.NEXUS_USERNAME }}
```
```
$ git grep maven-username 
.github/actions/maven-deploy-file/action.yml:  maven-username:
.github/actions/maven-deploy-file/action.yml:        MAVEN_USERNAME: ${{ inputs.maven-username }}
.github/actions/maven-deploy-file/action.yml:            echo "::error ::You need to provide credentials as maven-username and maven-password inputs when not providing a custom settings.xml"
.github/actions/maven-deploy-file/action.yml:        MAVEN_USERNAME: ${{ inputs.maven-username }}
```

Reply:
made the change nevertheless